Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -159,6 +164,43 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._repeatCountElement.textContent = Number.abbreviate(count);
     }
 
+    set timestamp(timestamp)
+    {
+        this._message._timestamp = timestamp;
```

Comment:
This breaks our general contract of now accessing private members of other classes. I know I previously commented that this view should just always get the timestamp from the message, but now that we are handling repeating messages, maybe this view should keep track of the timestamp like it does the repeat count.

Or if you keep the updating of the underlying WI.ConsoleMessage, you should add a `set timeout(newTimeout)` to `ConsoleMessage.js`